src/Models/AZ3166Device.ts

```diff
@@ -147,11 +143,22 @@ export class AZ3166Device extends ArduinoDeviceBase {
       throw new Error('Unable to find the board in the config file.');
     }
 
+    const plat = os.platform();
+
     this.generateCommonFiles();

```

Comment:
Consider using the `IoTWorkbenchProjectBase.generateIotWorkbenchProjectFile()`
Notice the different path for Workspace / Container project's `.iotworkbenchprojectfile`.